index.bs

```diff
@@ -162,7 +162,11 @@ If possible for the environment, clear the console. Otherwise, do nothing.
 
 <h4><dfn method for="Console">count(<var>label</var>)</dfn></h4>
 
-Counts the number of times count has been called with the provided <var>label</var>.
+<ol>
+  <li>Let <var>called</var> be the number of times <var>count</var> has been invoked (including this invocation) with the provided <var>label</var>.</li>
+  <li>Let <var>concat</var> be the concatenation of <var>label</var>, a colon (ASCII hex: 3A), a space (ASCII hex: 20), and <var>called</var>.
```

Comment:
For consistency with other specs, and some minor precision improvement, I'd phrase this as:

... the concatenation of _label_, U+003A COLON (:), U+0020 (SPACE), and ToString(_called_).

Also, this is minor but if you use `<emu-alg>` instead of `<ol>` you can use markdown-esque syntax for vars and list items. See other examples earlier in the file.